src/js/components/products/detail.tsx

```diff
@@ -40,25 +39,27 @@ const ProductDetail = ({ product, productSlug }: Props) => {
               <BreadCrumb
                 assistiveText={{ label: 'Two item breadcrumb' }}
                 trail={[
-                  <Link to={routes.home()}>Home</Link>,
+                  <Link to={routes.home()}>{i18n.t('Home')}</Link>,
                   <Link
                     className="slds-text-link_reset"
                     to={routes.product_detail(productSlug)}
                   >
-                    {product.name}
+                    {i18n.t(product.name)}
                   </Link>,
                 ]}
               />
             </div>
             <div className="slds-col slds-size_1-of-3">
               <h2 className="slds-m-top_large slds-m-bottom_small slds-text-heading_small">
-                [{product.name}]
+                [{i18n.t(product.name)}]
               </h2>
               <p
                 className="markdown slds-p-bottom_small"
                 dangerouslySetInnerHTML={{ __html: product.description }}
               />
-              <a href={product.repo_url}>Link to Github Repo</a>
+              <a href={product.repo_url}>
+                <Trans i18nKey="githubRepoLink">Link to Github Repo</Trans>
```

Comment:
There's some "missing translations" errors.  Is there an additional step to add translations?  Do we commit the changes to `/locales_dev/en/translation.json`

Reply:
Nevermind, about translations, looks like that is covered [here in the read.me](https://github.com/SFDO-Tooling/MetaShare/blob/master/CONTRIBUTING.rst#internationalization)